nuxeo-common/src/test/java/org/nuxeo/common/utils/TestDurationUtils.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Florent Guillaume
+ */
+package org.nuxeo.common.utils;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.time.Duration;
+import java.time.format.DateTimeParseException;
+
+import org.junit.Test;
+
+public class TestDurationUtils {
+
+    protected static final Duration parse(String s) {
```

Comment:
Is there an interest for static?

Reply:
It's an indirection for a static method... But I could go either way. I'll leave it as is though as it doesn't change the test.